force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -252,11 +337,19 @@ public with sharing class ClassesObjectsMethods {
      */
     public static String createAndGetBookDetails() {
         // Create a Book object with title "A Brief History of Time" and author "Stephen Hawking"
-
+        Book newbook  = new Book('A Brief History of Time' ,'Stephen Hawking');
+        newBook.Title = 'A Brief History of Time';
+        newBook.Author = 'Stephen Hawking';
+        
```

Comment:
You are repeating yourself here. you already set these values in the constructor